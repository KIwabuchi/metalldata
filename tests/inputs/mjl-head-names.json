{"_state": {"metall_location": "/PATH/TO/DATASTORE/m_names"}}
