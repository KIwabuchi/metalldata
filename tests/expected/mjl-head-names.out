{"returns":[{"id":1,"name":"Tim","home":3},{"id":2,"name":"Kim","home":5},{"id":3,"name":"Pam","home":6},{"id":4,"name":"Tom","home":5},{"id":5,"name":"Pat","home":9}]}
